lib/webdriver.py

```diff
@@ -121,35 +126,56 @@ def get_reservations(self, account_monitor: AccountMonitor) -> List[JSON]:
         # instead of requesting again later
         reservations = self._fetch_reservations(driver)
 
-        driver.quit()
+        self._quit_driver()
         return reservations
 
     def _get_driver(self) -> Driver:
+        # This environment variable is set in the Docker image
+        is_docker = os.environ.get("AUTO_SOUTHWEST_CHECK_IN_DOCKER") == "1"
+
         logger.debug("Starting webdriver for current session")
         browser_path = self.checkin_scheduler.reservation_monitor.config.browser_path
 
-        driver_version = "mlatest"
-        if os.environ.get("AUTO_SOUTHWEST_CHECK_IN_DOCKER") == "1":
-            # This environment variable is set in the Docker image. Makes sure a new driver
-            # is not downloaded as the Docker image already has the correct driver
-            driver_version = "keep"
-
-        driver = Driver(
-            binary_location=browser_path,
-            driver_version=driver_version,
-            headless=True,
-            uc_cdp_events=True,
-            undetectable=True,
-            is_mobile=True,
-        )
+        # Create a new temporary directory for Chrome profile
+        temp_dir = tempfile.mkdtemp()
+
+        driver_options = {
+            "binary_location": browser_path,
+            "user_data_dir": temp_dir,
+            "page_load_strategy": "none",
+            "uc_cdp_events": True,
+            "undetectable": True,
+            "incognito": True,
+        }
+
```

Comment:
Could we inline the `is_docker` variable so the if statement is ` if os.environ.get(…) == "1"`? The declaration is a bit separated from where it’s used, making it less readable.

Reply:
Yes I can make that change

Reply:
please review latest commit @jdholtz